frontend/src/app/main/ui/ds/notifications/toast.cljs

```diff
@@ -9,37 +9,33 @@
    [app.common.data.macros :as dm]
    [app.main.style :as stl])
   (:require
+   [app.common.data :as d]
    [app.main.ui.ds.foundations.assets.icon :as i]
+   [app.main.ui.ds.notifications.shared.notification-pill :refer [notification-pill*]]
    [rumext.v2 :as mf]))
 
-(def ^:private icons-by-level
-  {"info" i/info
-   "warning" i/msg-neutral
-   "error" i/delete-text
-   "success" i/status-tick})
-
 (def ^:private schema:toast
   [:map
    [:class {:optional true} :string]
-   [:level {:optional true}
-    [:maybe [:enum "info" "warning" "error" "success"]]]
+   [:type  {:optional true} [:maybe [:enum :toast :context]]]
+   [:level {:optional true} [:maybe [:enum :info :warning :error :success]]]
    [:on-close {:optional true} fn?]])
 
 (mf/defc toast*
   {::mf/props :obj
    ::mf/schema schema:toast}
-  [{:keys [class level children on-close] :rest props}]
-  (let [class (dm/str (stl/css-case :toast true
-                                    :toast-info (= level "info")
-                                    :toast-warning (= level "warning")
-                                    :toast-error (= level "error")
-                                    :toast-success (= level "success")) " " class)
-        icon-id (or (get icons-by-level level) i/msg-neutral)
-        props (mf/spread-props props {:class class})]
-    [:> "aside" props
-     [:*
-      [:> i/icon* {:icon-id icon-id :class (stl/css :icon)}]
-      children
+  [{:keys [class level type children on-close] :rest props}]
+  (let [class (dm/str class " " (stl/css-case :toast true))
+        level (if (string? level)
+                (keyword level)
+                (d/nilv level :info))
+        type (or type :context)
+        props (mf/spread-props props {:class class :role "alert"})]
+    [:> "div" props
```

Comment:
`<aside>`

Reply:
I changed it because in the [WAI ARIA patterns it recommends using a div element](https://www.w3.org/WAI/ARIA/apg/patterns/alert/examples/alert/)

Are you sure you want me to change it to `aside`?

Reply:
Yes, from the WAI ARIA patterns website:

> The code in this example is not intended for production environments.
> [...]
> - Robust accessibility can be further optimized by choosing implementation patterns that [maximize use of semantic HTML](https://www.w3.org/TR/using-aria/#rule1) and heeding the warning that [No ARIA is better than Bad ARIA](https://www.w3.org/WAI/ARIA/apg/practices/read-me-first/#no_aria_better_bad_aria).

The `<aside>` element is used for content that is separated from the main content. Here the separation it's clear (the toasts live independently and are ephemeral). 

I also have doubts that we should have the `alert` aria role for non-error toasts, but I'll leave that to your own criteria.

Reply:
The role alert is required, otherwise users with assistive technologies won't access to the information on the toast, while others do.

I'll make the changes

Reply:
I mentioned the role because of [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role):

<img width="816" alt="Screenshot 2025-02-11 at 9 32 58 AM" src="https://github.com/user-attachments/assets/3307ca48-e783-46bb-90ff-4ad600a02c36" />

Not all toasts are "urgent" (that's why we have different levels), and there are [other roles for live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes) as well. 

That said, if you truly think `alert` is the most appropriate for all our toasts, that's OK. But if so, I would make the politeness setting explicit and make non-errors be less intrusive than error toasts.